### PR TITLE
update agent to use jdk11 and supported Ubuntu LTS

### PR DESCRIPTION
**Pull request by jtnord**
Given we are telling users to run Java 11 it makes sense we use it for some ATH tests.

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/JavaGitContainer/Dockerfile [hunk 1]
```diff
@@ -3,15 +3,16 @@
 # and prepares for execution of Java compile jobs on slaves.
 #
 
-FROM ubuntu:xenial
+FROM ubuntu:jammy
```
Comment (timja) [inline src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/JavaGitContainer/Dockerfile:6]: any thoughts on using the version number instead which is easier than remembering each codename or googling it?
```suggestion
FROM ubuntu:22.04
```

**Review by timja — APPROVED** [review]

File: src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/JavaGitContainer/Dockerfile (hunk 1 above)
Comment (jtnord) [reply src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/JavaGitContainer/Dockerfile:6]: I have no objection

**Comment by jtnord** [comment]
Update center issue?

Or detached plugins and plugin installation order.  Can't see why this worked before and not now unless it is some detached issue and a `withPlugin` installing something that needs it later.

Need to investigate tomorrow.

```
master61279|java.io.IOException: Failed to load: JAXB plugin (jaxb 2.3.6-1)
master61279| - Update required: JavaBeans Activation Framework (JAF) API (javax-activation-api 1.2.0-2) to be updated to 1.2.0-3 or higher
master61279|	at hudson.PluginWrapper.
```

**Comment by timja** [comment]
I assume related to https://github.com/jenkinsci/jaxb-plugin/releases/tag/jaxb-2.3.6-1 and https://github.com/jenkinsci/jaxb-plugin/pull/23

**Comment by jtnord** [comment]
Released 8 days ago. The build of the first commit should have failed?

**Pull request merged**